user-manager/service/src/main/resources/service.properties

```diff
@@ -52,6 +52,6 @@ authentication-backend=internal
 # By default, be permissive with how users can use their Access Keys: allow them to access all services
 auth.access-key.create-new-session-context-if-necessary=true
 # By default, interactive users must approve new session context (i.e. approve each service they want to access)
```

Comment:
Just needs the comment language fixed (since flag is now the inverse of what it describes)